build/crd-sync.sh

```diff
@@ -23,6 +23,7 @@ generate_v1beta1() {
     yq '.spec.additionalPrinterColumns[] |= .JSONPath = .jsonPath' -i ${CRD_PATH}
     yq 'del(.spec.additionalPrinterColumns[].jsonPath)' -i ${CRD_PATH}
     yq '.spec.validation = .spec.versions[].schema' -i ${CRD_PATH}
+    yq '.spec.subresources = {"status": {}}' -i ${CRD_PATH}
```

Comment:
(Optional) I think this is a bit cleaner and more yaml-y.

```suggestion
    yq '.spec.subresources.status = {}' -i ${CRD_PATH}
```

Reply:
I had also noticed that the upgrade removed the `status` from the CRD itself. I'm assuming that's okay since I think it was populated with default values:
```
status:
  acceptedNames:
    kind: ""
    plural: ""
  conditions: []
  storedVersions: []
```
ref: https://github.com/open-cluster-management-io/governance-policy-addon-controller/commit/b20257a9b2912d0b1fae0b71d399927577a7cb04#diff-baaa7ee51809a08350650c5ff886e2df8b6e433101f984cc48516a594cd6e3c6L276-L281

Reply:
That status field shouldn't be there as it gets set by Kubernetes. Here is what it looks like on OCP 3.11 with this change:

```yaml
status:
  acceptedNames:
    kind: Policy
    listKind: PolicyList
    plural: policies
    shortNames:
      - plc
    singular: policy
```